server/views/pages/overview/partials/supportNeedsTab/supportNeedsTabContents.njk

```diff
@@ -9,57 +17,60 @@
-                {{ healthAndSupportNeed.prisonName }}
-              </h3>
-              {% if healthAndSupportNeed.hasSupportNeeds %}
-                <dl class="govuk-summary-list govuk-!-margin-bottom-6" data-qa="support-needs-list">
-                  <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                      Rapid screener from induction
-                    </dt>
-                    <dd class="govuk-summary-list__value" data-qa="rapid-assessment-date">
-                      {{ healthAndSupportNeed.rapidAssessmentDate | formatDate('D MMMM YYYY') | fallbackMessage('Not recorded in Curious') }}
-                    </dd>
-                  </div>
-                  <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                      In-depth assessment by education
-                    </dt>
-                    <dd class="govuk-summary-list__value" data-qa="in-depth-assessment-date">
-                      {{ healthAndSupportNeed.inDepthAssessmentDate | formatDate('D MMMM YYYY') | fallbackMessage('Not recorded in Curious') }}
-                    </dd>
-                  </div>
-                  <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                      Primary area of need identified
-                    </dt>
-                    <dd class="govuk-summary-list__value" data-qa="primary-ldd-needs">
-                      {{ healthAndSupportNeed.primaryLddAndHealthNeeds | fallbackMessage('Not recorded in Curious') }}
-                    </dd>
-                  </div>
-                  <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                      Other areas of need identified
-                    </dt>
-                    <dd class="govuk-summary-list__value" data-qa="additional-ldd-needs">
-                      {% if healthAndSupportNeed.additionalLddAndHealthNeeds.length > 0 %}
-                        <ul class="govuk-list">
-                          {% for additionalLddAndHealthNeed in healthAndSupportNeed.additionalLddAndHealthNeeds %}
-                            <li>{{ additionalLddAndHealthNeed }}</li>
-                          {% endfor %}
-                        </ul>
-                      {% else %}
-                        Not recorded in Curious
-                      {% endif %}
-                    </dd>
-                  </div>
-                </dl>
-              {% else %}
-                <p class="govuk-body" data-qa="no-screener-for-prison-message">No screener and assessment results recorded in Curious.</p>
-              {% endif %}
+            {% for lddAssessment in lddAssessments %}
```

Comment:
The nunjucks template looks like it's changed a lot, but it hasnt really.
The changes here are:
* Use the new wrapped promises for supportNeeds and prisonNamedById
* Work out whether there is an LDD assessment recorded in at least one prison (rather than it being in the controller above)
* Rename the loop variables to better reflect their purpose
* Wrap each loop iteration in a html `<section>` so that each are semantically separate and we can target/identify them better in the unit tests

Because of the addition of the `<section>` everything has been indented by 1 more step, hence it looking like a bigger change than it really is